src/styles/blocks/page.scss

```diff
@@ -0,0 +1,68 @@
+.page {
+  background-color: $black-color;
+
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  color: $white-color;
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+
+    transform: translateX(100%);
+    transition-property: transform;
+
+    &:target {
+      transform: translateX(0);
+    }
+  }
+
```

Comment:
```suggestion

&--with-menu {
  overflow: hidden;
}

```
scroll are not hidden when menu opened
![image](https://user-images.githubusercontent.com/98812993/202847647-371b9233-b7fc-49ea-ba0a-6cae3fd426b1.png)